tools/wpt/browser.py

```diff
@@ -35,6 +35,10 @@ def install(self, dest=None):
     def install_webdriver(self):
         return NotImplemented
 
+    @abstractmethod
+    def find_webdriver(self):
```

Comment:
With no other changes to `find_webdriver` in this PR, an explanation of why this change is needed in the commit message would be good. Does this mean that a subclass not implementing this will blow up? (You can write a block in a comment describing what the commit message should be when merging.)

Reply:
Actually, let me hack the history of this branch. I'll make a feature change commit + a code health commit. And then we can rebase-merge this PR.